torch/_jit_internal.py

```diff
@@ -421,18 +445,36 @@ def fn(*args, **kwargs):
         else:
             return if_false(*args, **kwargs)
 
-    if if_true.__doc__ is None and if_false.__doc__ is not None:
-        doc = if_false.__doc__
-        if_true.__doc__ = doc
```

Comment:
Not sure why we set the docstring also on the other dispatched function here. Is there a use case for this? I can only imagine this being useful, if the dispatched function also need to be callable on their own.

Reply:
No clue, maybe @driazati knows?

Reply:
Right, the functions are supposed to be "the same" except for the return type, so this would ensure that `help(fun)` still worked since the dispatched functions are usually public